services/cinema_hall.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> models.QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(name: str, rows: int, seats_in_row: int) -> CinemaHall:
+    return CinemaHall.objects.create(
+        name=name,
+        rows=rows,
+        seats_in_row=seats_in_row

```

Comment:
There is a typo in the parameter name `seats_in_row`. It should be `seats_in_rows` to match the field name in the `CinemaHall` model.